stable/datadog/values.yaml

```diff
@@ -47,6 +47,12 @@ daemonset:
 deployment:
   enabled: false
   replicas: 1
+  # Affinity for pod assignment
+  # Ref: https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity
+  # affinity: {}
+  # Tolerations for pod assignment
+  # Ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+  # tolerations: []
```

Comment:
Same for this one. As we don't do it in other Charts either, even though the value is empty.